### PR TITLE
add repos api

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

File: springboard_iogt/utils.py [hunk 1]
```diff
@@ -100,6 +101,18 @@ def known(cls, indexes, localizer=None):
                 if cls.exists(section.get('name'), indexes)]
 
     @classmethod
+    def get_index(cls, name, indexes):
+        [index] = [index for index in indexes if name in index]
```
Comment (justinvdm) [inline springboard_iogt/utils.py]: Can we be sure there will only ever be one result here?

**Comment by justinvdm** [comment]
:+1: left a minor comment, but probably shouldn't delay this PR landing.

File: springboard_iogt/utils.py (hunk 1 above)
Comment (miltontony) [inline springboard_iogt/utils.py]: No, I was going fix this -  I worked on this before getting on the plane - it needs to handle errors better here. If the index is missing, you're going to have problems.

**Comment by miltontony** [comment]
@justinvdm I'm happy for you to take over this PR - I won't be able to push it to the finish line

File: springboard_iogt/utils.py (hunk 1 above)
Comment (justinvdm) [inline springboard_iogt/utils.py]: I'm a bit confused. I would have expected the `if cls.exists(section.get('name'), indexes)` to cover cases where the index doesn't exist in `known_indexes()`. Was wondering more about cases where we have multiple indices that match `name in index`.

I don't think I have enough context to know whether the API should be returning an error for these cases, or whether it is not an error case if there are 0 or multiple indices matching a section name. I'm going to assume the latter for now, and make sure those cases are handled.

**Comment by justinvdm** [comment]
@saeedmb8 ready for review.

**Comment by miltontony** [comment]
:+1:

**Pull request merged**